docs/source/cli_help/discover.rst

```diff
@@ -9,7 +9,7 @@ discover
     git machete discover [-C|--checked-out-since=<date>] [-l|--list-commits] [-r|--roots=<branch1>,<branch2>,...] [-y|--yes]
 
 Discovers and displays tree of branch dependencies using a heuristic based on reflogs and asks whether to overwrite the existing definition :ref:`file` with the new discovered tree.
-If confirmed with a ``y[es]`` or ``e[dit]`` reply, backs up the current definition file (if it exists) as ``$GIT_DIR/machete~`` and saves the new tree under the usual ``$GIT_DIR/machete`` path.
+If confirmed with a ``y[es]`` or ``e[dit]`` reply, backs up the current definition file (if it exists) as ``machete~`` and saves the new tree in the usual ``machete`` file.
```

Comment:
TBH not sure if this is more clear than the original

Reply:
Okay, lemme restore the reference to `$GIT_DIR` (even though it's a mental shortcut)